tutorials/inputs/input_examples.rst

```diff
@@ -102,6 +131,17 @@ avoid this, make sure to test the event type first:
         if event is InputEventMouseButton:
             print("mouse button event at ", event.position)
 
+ .. code-tab:: csharp
+
+    // input event - runs when the input happens
+    public override void _Input(InputEvent inputEvent)
+    {
+        var mouseEvent = inputEvent as InputEventMouseButton;
+        if (mouseEvent != null)
```

Comment:
Use the shorthand for type checking with casting via `is`:

    if (inputEvent is InputEventMouseButton mouseEvent)